metadata.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+name: route53-lego-operator
+
+display-name: Route53 Lego Operator
+
+description: |
+  LEGO operator implementing the provider side of the `tls-certificates`
```

Comment:
In general try to say ACME instead of LEGO. LEGO is just an implementation of the ACME protocol and could be changed in the future. The same goes with the charm name. It should contain ACME, not LEGO.

Reply:
you are right, the `Lego` specific logic had been moved to the `acme_client` library.
I changed it :) and I will open a PR in `namecheap-lego-operator` targeting the same issue.